### DISCUSSION TITLE
Not showing any code in repositories - also of my own

### DISCUSSION THREAD

**Discussion by omertzuk** [community/community#86584] [Repositories] [answered]
### Select Topic Area

Bug

### Body

Somehow I don't see any code in any of the repositories I have on GitHub, and when I check other repositories I also don't see code... what could be the reason?
Here is one of my repositories - https://github.com/Omer80/DatasetsComplexNetworksCourse
![image](https://github.com/community/community/assets/3463905/5c80806c-9c2a-4dd0-b93c-68e8c5d78c11)

**Comment by PrashantRatanparikhe** [comment]
Hello @Omer80 
I just open your [repository](https://github.com/Omer80/DatasetsComplexNetworksCourse) and able to see your code.

![image](https://github.com/community/community/assets/11390157/9167fab1-f414-4e98-a533-91a4537b77a9)

Thanks

**Reply by ityashag** [reply]
![image](https://github.com/community/community/assets/97851640/7d915180-42db-4ff2-8690-7dc29582b54c)
there is code

**Reply by omertzuk** [reply]
Strange thing... I enter any repository through Firefox and I don't see any code, but if I entered through Google Chrome I do see the code. Here is an [example of a repository I tried to enter through Firefox](https://github.com/memgraph/odin), this is what I see - 
<img width="1355" alt="Screenshot 2024-01-13 at 15 18 32" src="https://github.com/community/community/assets/3463905/126d3853-3758-4dc6-82dc-03c869b17ac5">

**Comment by ityashag** [comment, marked as answer]
It might be due to browser-specific compatibility issues or cache problems.
Try clearing your Firefox cache or accessing GitHub in a private browsing
window to see if the issue persists. If the problem persists, check if
there are any browser extensions affecting GitHub's display.

**Reply by omertzuk** [reply]
Clearing Firefox cache solved the problem... first time it happened to me... thanks!

**Reply by m3047** [reply]
Clearing all data doesn't fix it for me. Bear in mind that it doesn't happen for me on most systems. I was able to reproduce from a report on an old Macbook Air running El Cap with both the original Safari and a fairly recent Firefox.

"Upgrade" is not a good answer for the poor and those who don't have first world problems.

**Comment by NS-Sheam** [comment]
<img width="699" alt="git2" src="https://github.com/community/community/assets/117335351/6e20cedf-f43a-4079-92e8-37ae6a2f7b8a">

I opened your repository and able to see code. You can try changing browser.

**Comment by m3047** [comment]
See also #86990

**Comment by omertzuk** [comment]
Issue solved!
